packages/snaps-rpc-methods/src/restricted/notify.ts

```diff
@@ -200,5 +271,28 @@ export function getValidatedParams(params: unknown): NotifyParams {
     });
   }
 
-  return params as NotificationArgs;
+  try {
+    const validatedParams = createUnion(
+      params,
+      NotificationParametersStruct,
+      'type',
+    );
+
+    validateTextLinks(validatedParams.message, isOnPhishingList, getSnap);
+
+    if (hasProperty(validatedParams, 'footerLink')) {
+      validateTextLinks(
```

Comment:
I don't understand this, are we supporting Markdown in `footerLink` as well as `href`? If not, we only need to validate the href

Reply:
I was checking for a scenario if someone did put markdown in the text. Not sure if that's something we're not supporting? If not, should I just throw if there's any markdown detected?

Reply:
Depends on how we are rendering the footerLink. I would assume that we are simply rendering it as a string since we don't need to support Markdown there?

Reply:
Oh yeah we're just rendering as string, i'll remove this check.